cmake/utils.cmake

```diff
@@ -309,3 +309,26 @@ endfunction(get_external_project_default_values)
 
 # Create one for all the project using the default list separator
 get_external_project_default_values(MAYAUSD_EXTERNAL_PROJECT_GENERAL_SETTINGS "$<SEMICOLON>")
+
+#
+# Adapted from peptide_compute_timestamps()
```

Comment:
This build is not a Peptide build, so peptide_compute_timestamps() is unavailable.  Simply copy-pasted the relevant code here.